JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -667,7 +667,12 @@ class AppInsightsTests extends TestClass {
 
                 // mock user agent
                 let originalUserAgent = navigator.userAgent;
-                this.setUserAgent("Googlebot/2.1");
+                try {
+                    this.setUserAgent("Googlebot/2.1");
+                } catch (ex) {
+                    Assert.ok(true, 'cannot run this test in the current setup - try Chrome');
+                    return;
```

Comment:
why do we need to catch and ok-return, instead of proceeding as if Beacon API is not supported?